akka-cluster/src/test/scala/akka/cluster/JoinConfigCompatCheckerSpec.scala

```diff
@@ -394,7 +427,7 @@ class JoinConfigCompatCheckerSpec extends AkkaSpec with ClusterTestKit {
 
     }
 
-    "NOT be allowed to re-join a cluster when one of the cluster required properties are not available on the joining side" taggedAs LongRunningTest in {
+    "NOT be allowed to re-join a cluster when one of the cluster required properties are not available on the joining side" taggedAs LongRunningTest ignore {
```

Comment:
change back to `in` or is there a reason to `ignore` now?

Reply:
Thanks for the catch 👍